libraries/Microsoft.Bot.Schema/Microsoft.Bot.Schema.csproj

```diff
@@ -25,6 +25,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="AdaptiveCards" Version="1.2.3" />
```

Comment:
If we don't add the Adaptive Cards into this project we would have to duplicate all of the typings.  Our API wants to return adaptive cards.  Going and duplicating the schema seems very wrong.  Did you have some sort of other suggestion?  Is there a reason we can't include the adaptivecards package?

Reply:
Primarily because it increases dependencies for every user of SDK, regardless of whether they use Sharepoint.  Typically something we attempt to avoid in an SDK.

Reply:
Is there a particular reason AdaptiveCards 1.2.3 is used?  As opposed to 2.x or 3.0?  I ask because we would need to provide parity in JavaScript.  3.0 is in beta for JS, so it would be of potential use. Though 2.x might be safer.